CONTRIBUTING.md

```diff
@@ -467,7 +467,7 @@ cargo build --release && hyperfine --warmup 10 \
   "./target/release/ruff check ./crates/ruff_linter/resources/test/cpython/ --no-cache -e --select W505,E501"
 ```
 
-You can run `poetry install` from `./scripts/benchmarks` to create a working environment for the
+You can run `uv venv --project ./scripts/benchmarks`, activate the venv and then run `uv venv --project ./scripts/benchmarks` to create a working environment for the
```

Comment:
These two commands are the same but the text says to run the first and then the second (same) command.